### PR TITLE
unixODBC, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: dev-db/unixODBC/unixodbc-2.3.4.recipe [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        settings/odbc.ini keep-old
+        settings/odbcinst.ini keep-old
+	"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	unixodbc${secondaryArchSuffix} = $portVersion
+	cmd:odbc_config${secondaryArchSuffix}
```
Comment (korli) [inline dev-db/unixODBC/unixodbc-2.3.4.recipe]: Question : is odbc_config meant to give libraries and headers path information? If yes, it belongs to the devel package.

Comment (Begasus) [inline dev-db/unixODBC/unixodbc-2.3.4.recipe]: I tried to add it to the devel package but ran into problems with it then ... let me check

Comment (Begasus) [inline dev-db/unixODBC/unixodbc-2.3.4.recipe]: Usage: odbc_config
                [--prefix]
                [--exec-prefix]
                [--include-prefix]
                [--lib-prefix]
                [--bin-prefix]
                [--version]
                [--libs]
                [--static-libs]
                [--libtool-libs]
                [--cflags]
                [--odbcversion]
                [--longodbcversion]
                [--odbcini]
                [--odbcinstini]
                [--header]
                [--ulen]

Comment (korli) [inline dev-db/unixODBC/unixodbc-2.3.4.recipe]: Then add $binDir/odbc_config after $developDir. It should be ok.

Comment (Begasus) [inline dev-db/unixODBC/unixodbc-2.3.4.recipe]: Warning: POLICY ERROR: "/packaging/unixodbc_devel/bin/odbc_config" needs library "libroot.so", but the package doesn't seem to declare that as a requirement

Comment (korli) [inline dev-db/unixODBC/unixodbc-2.3.4.recipe]: You need to add haiku as a requirement of the devel package.

**Comment by Begasus** [comment]
This should do it ... (build for x86_gcc2 just now)

**Pull request merged**